### PR TITLE
Add the /runningpods/ api endpoint

### PR DESCRIPTION
**Pull request by sargun**
This adds an API endpoint from the kubelet (/runningpods/). It is
an endpoint on kubelet which is considered a "debug" endpoint, so
it might be worth exposing through the options, but by default
it is exposed in most k8s configs AFAICT.

### CODE REVIEW

File: vkubelet/api/pods.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	scheme := runtime.NewScheme()
+	v1.SchemeBuilder.AddToScheme(scheme)
+	codecs := serializer.NewCodecFactory(scheme)
+
+	return handleError(func(w http.ResponseWriter, req *http.Request) error {
+		ctx := req.Context()
+		ctx = log.WithLogger(ctx, log.L)
+		pods, err := p.GetPods(ctx)
+		if err != nil {
+			return strongerrors.System(err)
```
Comment (sargun) [inline vkubelet/api/pods.go]: @cpuguy83 No idea if this is the right one to use.

Comment (cpuguy83) [reply vkubelet/api/pods.go]: The provider should give the correct error rather than try to figure it out here.

Comment (sargun) [reply vkubelet/api/pods.go]: Will handlError handle it correctly if they don't return a strongerror?

**Comment by sargun** [comment]
@cpuguy83 any opinions?

**Pull request merged**